app/controllers/organizations/adopter_fosterer/external_form_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Organizations
+  module AdopterFosterer
+    class ExternalFormController < ApplicationController
+      skip_verify_authorized # MARK: what should the auth policy look like for this action?
+
+      before_action :form_instructions
+
+      layout :form_layout
+
+      def index
+        @form_url = Current.organization.external_form_url
+      end
+
+      private
+
+      def form_layout
+        return "adopter_foster_dashboard" if params[:dashboard]
+
+        "application"
+      end
+
+      def form_instructions
```

Comment:
I think it might cleaner to set an instance variable here which you can use to show the correct text in the view, rather than define the text in the controller. The text is really view layer logic. So, in the controller you can do something like 

```
@dashboard = params[:dashboard].present?
```

and in the view
```
if @dashboard
this text
else
that text
end
```